### PR TITLE
print disjuncts in Gentzen sequent separately

### PR DESCRIPTION
**Pull request by kroening**
- [ ] Each commit message has a non-empty body, explaining why the change was made.
- [X] My contribution is formatted in line with CODING_STANDARD.md.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [X] My PR is restricted to a single feature or bugfix.
- [X] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: src/goto-symex/show_vcc.cpp [hunk 1]
```diff
@@ -83,9 +83,22 @@ void show_vcc_plain(
...
+    if(s_it->cond_expr.id() == ID_or)
+      disjuncts = to_or_expr(s_it->cond_expr).operands();
+    else
+      disjuncts.push_back(s_it->cond_expr);
+
+    std::size_t count = 1;
+    for(const auto &disjunct : disjuncts)
+    {
+      std::string string_value =
+        from_expr(ns, s_it->source.pc->function, disjunct);
```
Comment (tautschnig) [inline src/goto-symex/show_vcc.cpp:98]: Aren't we removing the `function` member?

**Review by tautschnig — CHANGES_REQUESTED** [review]
1) Why?
2) Missing tests.
3) This piece of code will require further work once the `function` member is gone from `instructiont` as noted below.

**Comment by kroening** [comment]
1. It avoids long lines, and matches with the usual definition of the Gentzen sequent.
2. Ok, will add some.
3. #2977 takes care of this!

**Review by tautschnig — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
This PR failed Diffblue compatibility checks (cbmc commit: 871298a721532c95a6662227f8bcf356246f50b2).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/87887369
Status will be re-evaluated on next push.
Please contact @peterschrammel, @thk123, or @allredj for support.

Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).

The incompatibility may have been introduced by an earlier PR. In that case merging this
PR should be avoided unless it fixes the current incompatibility.

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: 09a79897669a40fcef310ff61802cf11805b0cb0).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/87893119

**Pull request merged**